### PR TITLE
Compute the V1 output format from TimelineCred

### PR DESCRIPTION
**Pull request by decentralion**
This commit builds on #1781, adding the logic for computing the first
output format from TimelineCred. See #1773 for context.

Test plan:
The logic is simple, but has a couple interesting edge cases. I've added
unit tests to cover them. `yarn test` passes.

### CODE REVIEW

File: src/analysis/output.js [hunk 1]
```diff
@@ -42,6 +47,32 @@ export type OutputV1 = {|
...
+  const nodeEvaluator = nodeWeightEvaluator(weights);
+  const orderedNodes = Array.from(graph.nodes()).map((n) => {
+    const {description, address, timestampMs} = n;
+    const cred = NullUtil.get(tc.credNode(address)).total;
+    // In TimelineCred, a node with a null timestamp will never mint cred, because we don't
+    // know what period to mint it in.
+    // When we transition to CredRank, we should remove this check.
+    const minted = timestampMs == null ? 0 : nodeEvaluator(address);
+    const timestamp =
+      timestampMs == null ? null : Timestamp.fromNumber(timestampMs);
```
[the remark this answers is not in the document]
Comment (decentralion) [reply src/analysis/output.js:64]: Strict equality isn't needed when null checking, unless one cares about the difference between `null` and `undefined`. In this case the flow type is `timestampMs: number | null`, and `0 !== null`, so we don't need strict checking. we have a lint rule that enforces strict checking for everything except null/undefined.

File: src/analysis/output.js [hunk 2]
```diff
@@ -42,6 +47,32 @@ export type OutputV1 = {|
   +plugins: $ReadOnlyArray<PluginDeclaration>,
 |};
 
+export function fromTimelineCredAndPlugins(
+  tc: TimelineCred,
+  plugins: $ReadOnlyArray<PluginDeclaration>
+): OutputV1 {
```
[the remark this answers is not in the document]
Comment (decentralion) [reply src/analysis/output.js:53]: Done

**Comment by decentralion** [comment]
@hammadj I addressed your comments. Would appreciate a nit-and-approve in cases like this, so that once I've addressed your questions I can easily merge it without another round of review. (Since the suggestions are pretty easy to follow thru on.)

**Comment by hammadj** [comment]
> @hammadj I addressed your comments. Would appreciate a nit-and-approve in cases like this, so that once I've addressed your questions I can easily merge it without another round of review. (Since the suggestions are pretty easy to follow thru on.)

Got it, will do next time!

**Comment by decentralion** [comment]
FYI @hammadj, I undid the change you suggested about factoring computing the output nodes into a separate function. As soon as I got around to adding the next part of the output format, I wound up needing the same variables split out, and needed to cache bits of compute from generating the output nodes.
